CHANGELOG

```diff
@@ -2,6 +2,12 @@
 # All notable changes to this project will be documented in this file.
 # This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [X.XX.X] XXXX-XX-XX
```

Comment:
We normally just do:

```
## [0.40.0] UNRELEASED
```

The major number change because of the new functionality.